docs/top.rst

```diff
@@ -54,6 +55,9 @@ This level enables fully connected multi-layer perceptron.
    nnvm.symbol.softmax
    nnvm.symbol.log_softmax
    nnvm.symbol.pad
+   nnvm.symbol.assign
+   nnvm.symbol.make_loss
```

Comment:
make loss seems to be not necessary for default gradient, i think we should avoid introduce this for now